docs/core.adoc

```diff
@@ -10,22 +10,34 @@ It's main features are:
 
 == Installation
 
-Use the Cordova CLI to add the Core module as a plugin to your project:
+The `@aerogear/core` module is available via NPM:
 
 ----
-cordova plugin add @aerogearservices/core-cordova
+npm install @aerogear/core
 ----
 
-This will install the package from the NPM registry and properly link all native dependencies to the project's cordova Platforms.
+=== Additional steps for a Cordova project
+
+After node module installation via NPM use the Cordova CLI to add the additional native dependencies to your project:
+
+----
+cordova plugin add cordova-plugin-device
+cordova plugin add cordova-plugin-app-version
+----
+
+This will install the packages with these native dependencies from the NPM registry and properly link all native dependencies to the project's cordova Platforms.
 
 Refer to the link:./getting-started[Getting Started] guide for more information on installing the AeroGear SDK.
 
 == Usage
 
-After installation, the Core module will become available through the global `window.aerogear` object in your project.
-
-=== Logging
+The Core module supports parsing of the configuration file generated by the mobile CLI, in practice this feature is used internally by other modules of the SDK:
 
-=== Networking
+[source, javascript]
+----
+import { MetricsService } from '@aerogear/core';
+import configuration from '../mobile-services.json';
 
-=== Metrics Service
+// utilizes Core to retrieve it's own configuration
+const metrics = new MetricsService(configuration);
```

Comment:
@josemigallas @wtrocki  I've tried to change the Usage section to point out how core's used by other modules and doesn't make sense by itself but it can be a bit clunky, removing this section is also an option.